Source/WebCore/platform/mac/ScrollbarsControllerMac.mm

```diff
@@ -957,8 +958,7 @@ - (void)invalidate
         auto newVerticalPainter = retainPtr([NSScrollerImp scrollerImpWithStyle:newStyle controlSize:(NSControlSize)verticalScrollbar->widthStyle() horizontal:NO replacingScrollerImp:oldVerticalPainter]);
```

Comment:
I think you should strive to only have one call site that creates `NSScrollerImp` objects. Having one here, and one in ScrollbarMac forces me to think about how the arguments differ.

Reply:
Right, changed this to use createScrollerImp and pass the old NSScrollerImp.